### ISSUE TITLE
App freezes when Adhan launches. (Linux only)

### ISSUE THREAD

**Issue by DBChoco** [DBChoco/Salawat#3]
If you're experiencing this, try to disable notifications, it'll take some time for me to patch this.

**Comment by LiverBlow** [comment]
Am seeing this issue too on PopOS, I'll use muezzin in the mean time.

**Comment by DBChoco** [comment]
> Am seeing this issue too on PopOS, I'll use muezzin in the mean time.

Yes sometimes it does do that, I've tried many things to fix it but just couldn't.
Unfortunately it seems like I'll have to wait for someone else more experienced to have a look.
